pkgs/tools/misc/logstash/6.x.nix

```diff
@@ -3,10 +3,14 @@
 , lib, stdenv
 , fetchurl
 , makeWrapper
-, jre
+, jdk11_headless
 }:
 
 with lib;
+let
+  # https://www.elastic.co/support/matrix#logstash-and-jvm
+  jre = jdk11_headless;
+in
```

Comment:
Do you know by chance when this approach and overring the input of jre is preferred?

Reply:
idk, how else would you do it?

Reply:
I typed a bit fast and did some typos. The alternative would be to assign jre11_headless to jre in top-level.nix for logstash.

Reply:
Right. I guess that would be easier to override.